libbeat/docs/outputconfig.asciidoc

```diff
@@ -177,6 +177,35 @@ The index root name to write events to. The default is the Beat name.
 For example "{beatname_lc}" generates "[{beatname_lc}-]YYYY.MM.DD" indexes (for example,
 "{beatname_lc}-2015.04.26").
 
+===== template
+
+The http://www.elastic.co/guide/en/elasticsearch/reference/current/indices-templates.html[index
+template] to use for setting mappings in Elasticsearch. By default, template loading is
+disabled. If you leave this option disabled, you must load the template by running the
+load script that is provided with {beatname_uc}. 
+
+If you enable the `template` config option, you can adjust the following settings to load your
+own template or overwrite an existing one:
+
+*`name`*:: The name of the template. The default is `packetbeat`.
```

Comment:
Good catch. I guess I shouldn't have tested this by building the Packetbeat docs. :-) Issue fixed.